sources/engine/Xenko.Graphics/UIBatch.cs

```diff
@@ -26,7 +26,7 @@ public class UIBatch : BatchBase<UIBatch.UIImageDrawInfo>
         /// <summary>
         /// The view projection matrix that will be used for the current begin/end draw calls.
         /// </summary>
-        private Matrix viewProjectionMatrix;
+        public Matrix viewProjectionMatrix;
```

Comment:
Please keep this private and pass it to `SpriteFont.InternalUIDrawGlyph` as a `ref Matrix`